test/assembler/encoder.test.ts

```diff
@@ -58,6 +58,36 @@ describe('encode', function () {
     expect(file.content[0].value).toBe(0x77)
     expect(file.content[1].value).toBe(0x44)
   })
+  it('should encode DCB instruction with Strings', function () {
+    const code: ICodeFile = {
+      symbols: {},
+      areas: [
+        {
+          type: AreaType.Data,
+          isReadOnly: true,
+          name: '|.data|',
+          instructions: [
+            {
+              name: 'DCB',
+              options: ['"TestString"', `"TestString with Escape'""`],
+              line: 0
+            }
+          ]
+        }
+      ]
+    }
+    const file = encode(code)
+    expect(Object.keys(file.sections).length).toBe(1)
+    expect(getSection(file, '|.data|').offset).toBe(0)
+    expect(getSection(file, '|.data|').size).toBe(33)
+    expect(file.content.length).toBe(33)
+    expect(file.content[0].value).toBe(84)
+    expect(file.content[1].value).toBe(101)
+    expect(file.content[9].value).toBe(103)
+    expect(file.content[19].value).toBe(103)
+    expect(file.content[25].value).toBe(32)
+    expect(file.content[32].value).toBe(34)
```

Comment:
check file.content[31] to be e and not single quote

Reply:
Done